graphdatascience/query_runner/arrow_endpoint_version.py

```diff
@@ -1,17 +1,35 @@
+from __future__ import annotations
+
 from enum import Enum
 
 
 class ArrowEndpointVersion(Enum):
     ALPHA = ""
     V1 = "v1/"
 
-    def name(self):
+    def name(self) -> str:
         return self._name_.lower()
 
-    def prefix(self):
+    def prefix(self) -> str:
         return self._value_
 
+    @staticmethod
+    def from_arrow_info(arrow_info: Series[Any]) -> ArrowEndpointVersion:
+        supported_arrow_versions = arrow_info.get("versions", [])
+        # Fallback for pre 2.6.0 servers that do not support versions
+        if len(supported_arrow_versions) == 0:
+            return ArrowEndpointVersion.ALPHA
```

Comment:
NIT: for empty list you can also do `if not supported_arrow_versions`

Reply:
yeah, I read about that, but preferred the more explicit way